test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -519,10 +515,9 @@ mod inputs {
                     .await
                     .unwrap();
                 let messages = wallet.get_messages().await?;
-                let inputs = wallet.get_inputs_for_messages(0).await?;
-                // let contract_bytes: [u8; 32] = result.value.into();
-                // let message_id: [u8; 32] = *inputs[0].message_id().unwrap().into();
-                // assert_eq!(result.value, Bits256(message_id));
+                let message_id: [u8; 32] = *messages[0].message_id.0 .0;
```

Comment:
What exactly is this doing? The space between the two `.0` is throwing me off...